pnpm/src/cmd/completion.ts

```diff
@@ -28,7 +29,7 @@ export function createCompletion (
     // We cannot autocomplete what a user types after "pnpm test --"
     if (inputArgv.includes('--')) return
     const { params, options, cmd } = await parseCliArgs(inputArgv)
-    return tabtab.log(
+    tabtab.log(
```

Comment:
Does tabtab.log write to the console itself?

Reply:
> Does tabtab.log write to the console itself?

Yes.